src/graph_compute/common.h

```diff
@@ -35,6 +35,10 @@
   #define WHEN_DEBUG(ex)
 #endif
 
+#ifndef NUM_CORES
+  #define NUM_CORES 12
```

Comment:
OK - I switched to just using NUMA_WORKERS.  I didn't like using something prefixed with 'NUMA' for anything other than D1_NUMA, but it's not a big deal.